src/referral_maintenance.rs

```diff
@@ -46,6 +46,10 @@ pub async fn maintenance_tasks(
 ) -> Result<()> {
     // 1st step: getting all possible app_data
     // 1.1: Load app_data from dune download
+    tracing::info!(
```

Comment:
I think its a great idea to introduce info messages for each maintance loop. But maybe its too much for every step. these logs could then be to debug logs.

Reply:
Unfortunately, the debug logs are too verbose to actually managed to see any of this content... I would prefer to keep them as info so they are not drowning in the hundreds of other debug logs here. I will however, combine the two logs for step 1.1 and 1.2 if that helps.